src/MutantRunResultMatcher.ts

```diff
@@ -12,8 +13,7 @@ export default class MutantRunResultMatcher {
         if (testResult.coverage) {
           let coveredFile = testResult.coverage[mutant.filename];
           if (coveredFile) {
-            covered = Object.keys(coveredFile.statementMap).some(statementId => coveredFile.s[statementId] !== 0
-              && this.mutantCovers(mutant, coveredFile.statementMap[parseInt(statementId)]))
+            covered = this.mutantCoversFile(mutant, coveredFile);
```

Comment:
I think we can do this in 2 steps. Find out the id of the 'smallest' statement which covers the mutant *once*. After that start checking the test results (the `s` collection of each coverage result). The id of the 'smallest' statement shouldn't change when looping over the testresults... or... do you see a scenario in which they do change?